csi.proto

```diff
@@ -495,16 +513,16 @@ message TopologyRequirement {
   //   {"region": "R1", "zone": "Z4"},
   //   {"region": "R1", "zone": "Z2"}
   // then the the SP SHOULD first attempt to make the provisioned volume
-  // accessible from "zone" "Z4" in the "region" "R1" and fall back to
-  // "zone" "Z2" in the "region" "R1" if that is not possible. If that
-  // is not possible, the SP may choose between either the "zone"
-  // "Z3" or "Z5" in the "region" "R1".
+  // or snapshot accessible from "zone" "Z4" in the "region" "R1" and
+  // fall back to "zone" "Z2" in the "region" "R1" if that is not
+  // possible. If that is not possible, the SP may choose between either
+  // the "zone" "Z3" or "Z5" in the "region" "R1".
   //
   // Example 3:
-  // Given a volume should be accessible from TWO zones (because an
-  // opaque parameter in CreateVolumeRequest, for example, specifies
-  // the volume is accessible from two zones, aka synchronously
-  // replicated), and
+  // Given a volume or snapshot should be accessible from TWO zones
+  // (because an opaque parameter in CreateVolumeRequest, for example,
+  // specifies the volume is accessible from two zones, aka
+  // synchronously replicated), and
```

Comment:
This comment says that specifying the volume is accessible from two zones is equivalent to synchronous replication.  But per the discussion in the meeting, I don't believe that is necessarily true - replication behavior is out of scope, e.g. implementers *may* replicate to satisfy accessibility requirements but are not required to.

Reply:
I agree with this. It's important to state that while snapshot replication MAY take place in order to satisfy accessibility requrements, clients MUST NOT assume that accessibility has any relation to replication, and if replication is desired, it should be requested in a different manner. Today, that means using opaque parameters to request replication.